Makefile

```diff
@@ -1,6 +1,7 @@
 ##################################################
 # Variables                                      #
 ##################################################
+VERSION_NUM    ?= 2.8.1
```

Comment:
What is this good for?

Reply:
for `make bundle`
`$(KUSTOMIZE) build config/manifests | operator-sdk generate bundle -q --overwrite --version $(VERSION_NUM) $(BUNDLE_METADATA_OPTS)`

Reply:
hmm, I'd really appreciate a solution where we don't hardcore the version.

Is this version needed for testing only?